diff --git a/src/gmp.c b/src/gmp.c
index e8c3e7940..297fe0e5d 100644
--- a/src/gmp.c
+++ b/src/gmp.c
@@ -103,7 +103,7 @@
 #include <assert.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <math.h>
 #include <netinet/in.h>
 #include <string.h>
diff --git a/src/gmp.h b/src/gmp.h
index 364f39f7a..b64f9d718 100644
--- a/src/gmp.h
+++ b/src/gmp.h
@@ -27,7 +27,7 @@
 #include "manage.h"
 #include "types.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <gnutls/gnutls.h>
 #include <gvm/util/serverutils.h>
 #include <sys/types.h>
diff --git a/src/gmp_base.h b/src/gmp_base.h
index 15db793ad..db78d6bd2 100644
--- a/src/gmp_base.h
+++ b/src/gmp_base.h
@@ -19,7 +19,7 @@
 #ifndef _GVMD_GMP_BASE_H
 #define _GVMD_GMP_BASE_H
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 /**
  * @brief A handle on a GMP parser.
diff --git a/src/gmp_configs.c b/src/gmp_configs.c
index a1be727f5..077f1a413 100644
--- a/src/gmp_configs.c
+++ b/src/gmp_configs.c
@@ -30,7 +30,7 @@
 #include "manage_configs.h"
 
 #include <assert.h>
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <stdlib.h>
 #include <string.h>
 
diff --git a/src/gmp_delete.h b/src/gmp_delete.h
index 0adb621a7..37e8b052d 100644
--- a/src/gmp_delete.h
+++ b/src/gmp_delete.h
@@ -21,7 +21,7 @@
 
 #include "gmp_base.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 void
 delete_start (const gchar *, const gchar *, const gchar **, const gchar **);
diff --git a/src/gmp_license.h b/src/gmp_license.h
index 4802fdacd..ea49d2778 100644
--- a/src/gmp_license.h
+++ b/src/gmp_license.h
@@ -18,7 +18,7 @@
 
 #include "gmp_base.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <gvm/util/xmlutils.h>
 
 
diff --git a/src/gmp_tickets.c b/src/gmp_tickets.c
index 5e1e9c928..c067f6339 100644
--- a/src/gmp_tickets.c
+++ b/src/gmp_tickets.c
@@ -28,7 +28,7 @@
 #include "gmp_get.h"
 #include "manage_tickets.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <stdlib.h>
 #include <string.h>
 
diff --git a/src/gmp_tls_certificates.c b/src/gmp_tls_certificates.c
index 402abc74e..125f0916e 100644
--- a/src/gmp_tls_certificates.c
+++ b/src/gmp_tls_certificates.c
@@ -29,7 +29,7 @@
 #include "gmp_get.h"
 #include "manage_tls_certificates.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <stdlib.h>
 #include <string.h>
 
diff --git a/src/gmpd.h b/src/gmpd.h
index 18eec688d..eac51aba8 100644
--- a/src/gmpd.h
+++ b/src/gmpd.h
@@ -27,7 +27,7 @@
 #include "manage.h"
 #include "types.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <gnutls/gnutls.h>
 #include <gvm/util/serverutils.h>
 #include <netinet/in.h>
diff --git a/src/gvmd.c b/src/gvmd.c
index fbbbe73f3..eb15de049 100644
--- a/src/gvmd.c
+++ b/src/gvmd.c
@@ -72,8 +72,8 @@
 #include <assert.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <glib.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <gnutls/gnutls.h>
 #include <grp.h>
 #include <netdb.h>
diff --git a/src/iterator.h b/src/iterator.h
index dd241abc5..f01ef3cda 100644
--- a/src/iterator.h
+++ b/src/iterator.h
@@ -29,7 +29,7 @@
 
 #include "lsc_crypt.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 /* Types. */
 
diff --git a/src/lsc_crypt.c b/src/lsc_crypt.c
index d8927d3c4..1be830751 100644
--- a/src/lsc_crypt.c
+++ b/src/lsc_crypt.c
@@ -23,8 +23,8 @@
  * This file provides support for encrypting LSC credentials.
  */
 
-#include <glib.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <stdarg.h>
diff --git a/src/lsc_crypt.h b/src/lsc_crypt.h
index 153170e74..06dbf9a30 100644
--- a/src/lsc_crypt.h
+++ b/src/lsc_crypt.h
@@ -24,7 +24,7 @@
 #ifndef _GVMD_LSC_CRYPT_H
 #define _GVMD_LSC_CRYPT_H
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 /* (Defined in gvmd.c) */
 extern int disable_encrypted_credentials;
diff --git a/src/lsc_user.c b/src/lsc_user.c
index 5d7e8b6ec..ff52d78c7 100644
--- a/src/lsc_user.c
+++ b/src/lsc_user.c
@@ -23,8 +23,8 @@
  * This file provides support for generating packages for LSC credentials.
  */
 
-#include <glib.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
diff --git a/src/lsc_user.h b/src/lsc_user.h
index cdde9ff81..469de32e9 100644
--- a/src/lsc_user.h
+++ b/src/lsc_user.h
@@ -24,7 +24,7 @@
 #ifndef _GVMD_LSC_USER_H
 #define _GVMD_LSC_USER_H
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 int
 lsc_user_keys_create (const gchar *, gchar **);
diff --git a/src/manage.c b/src/manage.c
index 6e9017ee8..6700cb92e 100644
--- a/src/manage.c
+++ b/src/manage.c
@@ -65,7 +65,7 @@
 #include <errno.h>
 #include <dirent.h>
 #include <fcntl.h>
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <gnutls/x509.h> /* for gnutls_x509_crt_... */
 #include <math.h>
 #include <locale.h>
diff --git a/src/manage.h b/src/manage.h
index 03a11b83d..aceb6c1e2 100644
--- a/src/manage.h
+++ b/src/manage.h
@@ -30,7 +30,7 @@
 #include "utils.h"
 
 #include <stdio.h>
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <gnutls/gnutls.h>
 
 #include <gvm/base/array.h>
diff --git a/src/manage_acl.h b/src/manage_acl.h
index 16977d4a5..352f61790 100644
--- a/src/manage_acl.h
+++ b/src/manage_acl.h
@@ -25,7 +25,7 @@
 #define _GVMD_MANAGE_ACL_H
 
 #include "manage_sql.h"
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 /**
  * @brief Generate SQL for user permission check.
diff --git a/src/manage_configs.c b/src/manage_configs.c
index f06f03f88..ebac2f8f9 100644
--- a/src/manage_configs.c
+++ b/src/manage_configs.c
@@ -30,7 +30,7 @@
 #include "utils.h"
 
 #include <errno.h>
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <gvm/util/fileutils.h>
 #include <stdlib.h>
 #include <string.h>
diff --git a/src/manage_get.h b/src/manage_get.h
index 09d56fb7e..2a22244f1 100644
--- a/src/manage_get.h
+++ b/src/manage_get.h
@@ -26,7 +26,7 @@
 
 #include "iterator.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 /**
  * @brief Command data for a get command.
diff --git a/src/manage_license.h b/src/manage_license.h
index eb0e54a59..77c39f162 100644
--- a/src/manage_license.h
+++ b/src/manage_license.h
@@ -23,7 +23,7 @@
  * Headers for non-SQL license information code for the GVM management layer.
  */
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 #ifdef HAS_LIBTHEIA
 #include <theia/client.h>
diff --git a/src/manage_migrators.c b/src/manage_migrators.c
index b3c3ee2e2..ca066d4b3 100644
--- a/src/manage_migrators.c
+++ b/src/manage_migrators.c
@@ -117,7 +117,7 @@
 
 #include <assert.h>
 #include <errno.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <sys/param.h>
diff --git a/src/manage_port_lists.h b/src/manage_port_lists.h
index 32c884327..8a98700da 100644
--- a/src/manage_port_lists.h
+++ b/src/manage_port_lists.h
@@ -21,7 +21,7 @@
 
 #include "manage.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 gboolean
 find_port_list (const char*, port_list_t*);
diff --git a/src/manage_report_formats.c b/src/manage_report_formats.c
index b5f0fe403..5f4f036c8 100644
--- a/src/manage_report_formats.c
+++ b/src/manage_report_formats.c
@@ -32,7 +32,7 @@
 
 #include <assert.h>
 #include <errno.h>
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <gvm/util/fileutils.h>
 #include <locale.h>
 #include <stdlib.h>
diff --git a/src/manage_report_formats.h b/src/manage_report_formats.h
index 5172fdaef..1dc60ae77 100644
--- a/src/manage_report_formats.h
+++ b/src/manage_report_formats.h
@@ -21,7 +21,7 @@
 
 #include "manage.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 gboolean
 find_report_format_with_permission (const char*, report_format_t*,
diff --git a/src/manage_sql.c b/src/manage_sql.c
index 88c0f37f7..404ca131f 100644
--- a/src/manage_sql.c
+++ b/src/manage_sql.c
@@ -52,7 +52,7 @@
 #include <ctype.h>
 #include <dirent.h>
 #include <errno.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <gnutls/x509.h>
 #include <malloc.h>
 #include <pwd.h>
diff --git a/src/manage_sql_configs.c b/src/manage_sql_configs.c
index 7b5faefcf..4e51e285e 100644
--- a/src/manage_sql_configs.c
+++ b/src/manage_sql_configs.c
@@ -32,7 +32,7 @@
 
 #include <assert.h>
 #include <errno.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <stdlib.h>
 #include <string.h>
 
diff --git a/src/manage_sql_port_lists.c b/src/manage_sql_port_lists.c
index 790b023f9..981321562 100644
--- a/src/manage_sql_port_lists.c
+++ b/src/manage_sql_port_lists.c
@@ -29,8 +29,8 @@
 #include "sql.h"
 
 #include <errno.h>
-#include <glib.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <strings.h>
diff --git a/src/manage_sql_report_formats.c b/src/manage_sql_report_formats.c
index 304d75931..3ab913ad0 100644
--- a/src/manage_sql_report_formats.c
+++ b/src/manage_sql_report_formats.c
@@ -31,8 +31,8 @@
 #include "utils.h"
 
 #include <errno.h>
-#include <glib.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <grp.h>
 #include <libgen.h>
 #include <limits.h>
diff --git a/src/manage_sql_report_formats.h b/src/manage_sql_report_formats.h
index c7dd4154a..f77c19b21 100644
--- a/src/manage_sql_report_formats.h
+++ b/src/manage_sql_report_formats.h
@@ -22,7 +22,7 @@
 #include "manage.h"
 #include "manage_sql.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 gboolean
 lookup_report_format (const char*, report_format_t*);
diff --git a/src/manage_sql_secinfo.c b/src/manage_sql_secinfo.c
index ac2a86f4f..7cf32df87 100644
--- a/src/manage_sql_secinfo.c
+++ b/src/manage_sql_secinfo.c
@@ -39,7 +39,7 @@
 #include <fcntl.h>
 #include <fnmatch.h>
 #include <ftw.h>
-#include <glib/gstdio.h>
+#include <glib-2.0/glib/gstdio.h>
 #include <math.h>
 #include <stdlib.h>
 #include <string.h>
diff --git a/src/manage_sql_secinfo.h b/src/manage_sql_secinfo.h
index f3e331b93..791081768 100644
--- a/src/manage_sql_secinfo.h
+++ b/src/manage_sql_secinfo.h
@@ -21,7 +21,7 @@
  * @brief Manager Manage library: SQL backend headers.
  */
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #ifndef _GVMD_MANAGE_SQL_SECINFO_H
 #define _GVMD_MANAGE_SQL_SECINFO_H
 
diff --git a/src/manage_utils.h b/src/manage_utils.h
index dcdba5837..b650bdd00 100644
--- a/src/manage_utils.h
+++ b/src/manage_utils.h
@@ -27,7 +27,7 @@
 /* For strptime in time.h. */
 #undef _XOPEN_SOURCE
 #define _XOPEN_SOURCE
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <libical/ical.h>
 #include <time.h>
 
diff --git a/src/sql.h b/src/sql.h
index ab73b9bc2..8a010ba13 100644
--- a/src/sql.h
+++ b/src/sql.h
@@ -27,7 +27,7 @@
 #include "iterator.h"
 #include "manage.h"
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 
 /* Helpers. */
 
diff --git a/src/sql_pg.c b/src/sql_pg.c
index c8094953e..41783a0fa 100644
--- a/src/sql_pg.c
+++ b/src/sql_pg.c
@@ -29,7 +29,7 @@
 #include <endian.h>
 #include <errno.h>
 #include <arpa/inet.h>
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <inttypes.h>
 #include <netinet/in.h>
 #include <postgresql/libpq-fe.h>
diff --git a/src/utils.h b/src/utils.h
index 999a128ee..dc5ec8331 100644
--- a/src/utils.h
+++ b/src/utils.h
@@ -24,7 +24,7 @@
 #ifndef _GVMD_UTILS_H
 #define _GVMD_UTILS_H
 
-#include <glib.h>
+#include <glib-2.0/glib.h>
 #include <gvm/util/xmlutils.h>
 #include <time.h>
 
